ewatercycle/forcing/_lisflood.py

```diff
@@ -0,0 +1,110 @@
+"""Forcing related functionality for lisflood"""
+
+from pathlib import Path
+from typing import Optional
+
+from esmvalcore.experimental import get_recipe
+
+from ..util import data_files_from_recipe_output, get_extents, get_time
+from ._default import DefaultForcing
+from .datasets import DATASETS
+
+
+class LisfloodForcing(DefaultForcing):
+    """Container for lisflood forcing data."""
+
+    # TODO check whether start/end time are same as in the files
+    def __init__(
+        self,
+        start_time: str,
+        end_time: str,
+        directory: str,
+        shape: str,
+        PrefixPrecipitation: Optional[str] = 'pr.nc',
```

Comment:
You have probably discussed this at length already, but why are you using the weird capitalization? It would make a more uniform impression to stick with the PEP8 naming convention.

Reply:
We're just exposing the model configuration settings. This is passed on directly to the lisflood configuration file that's automatically created. If we change it here, then there will be a mapping from 'ewatercycle model settings' to 'native hydrological model settings'. I think it's better to stick with the native model options for now.